### PR TITLE
Add ability to set icon size

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Fixes: #874 

### How to review 
- Test small and large icons
- Make sure all other icons still display correctly

### CODE REVIEW

**Comment by rmccar** [comment]
[Issue #874: Need to be able to define a panel icon size](https://trello.com/c/fk5hduP7/611-issue-874-need-to-be-able-to-define-a-panel-icon-size)

**Review by boxadesign — APPROVED** [review]

**Review by jrbarnes9 — COMMENTED** [review]
Would be really useful if these icon sizes matched up to/extended the heading sizes (default, 'm', 'l', 'xl') and the vertical alignment and padding was adjusted accordingly...

Screenshot 2020-09-17 at 15 26 49

File: src/components/panel/_macro-options.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+| titleTag   | string  | false    | The html tag to wrap the title text in. Will default to a `div`                                       |
+| type       | string  | false    | The type of panel to render. Available options are `success`, `warn`, `error` and `branded`           |
+| spacious   | boolean | false    | Will render a more spacious version of the panel if set to `true`                                     |
+| classes    | string  | false    | Custom classes to add to the panel                                                                    |
+| id         | string  | false    | Custom id to add to the panel                                                                         |
+| attributes | object  | false    | HTML attributes to apply to the panel (e.g. data attributes)                                          |
+| icon       | string  | false    | Set this to the name of the icon you want to be included before the contents of the panel             |
+| iconSize   | string  | false    | Set this to the size of the icon you want can be set to "m", "l" "xl". If not set will defualt to "m" |
```
Comment (jrbarnes9) [inline src/components/panel/_macro-options.md:12]: Think the default needs to be the smallest icon (to match height of base font size), then the settings m, l, and xl match the three next upper font sizes

Comment (jrbarnes9) [reply src/components/panel/_macro-options.md:12]: I've pushed up a temporary success panel examples as there's still alignment and size issues.

Screenshot 2020-09-18 at 10 37 36

**Review by jrbarnes9 — CHANGES_REQUESTED** [review]

File: src/components/panel/_macro-options.md (hunk 1 above)
Comment (rmccar) [reply src/components/panel/_macro-options.md]: Ive now sorted this and added spacing between the text and icon for larger icon sizes

**Review by jrbarnes9 — APPROVED** [review]

**Pull request merged**